source_modelling/sources.py

```diff
@@ -610,16 +675,18 @@ def wgs_depth_coordinates_to_fault_coordinates(
 
         """
         # the right edges as a cumulative proportion of the fault length (e.g. [0.1, ..., 0.8])
-        right_edges = self.lengths.cumsum() / self.length
-        right_edges = np.append(right_edges, 1)
+        left_edges = self.lengths.cumsum() / self.length
```

Comment:
comment should probably change to left edge now

Reply:
ye